lib/new-admin/graphql/schema.js

```diff
@@ -222,18 +247,28 @@ const typeDefs = gql`
   }
 
   type Mutation {
-    machineAction(deviceId:ID!, action: MachineAction!, cassette1: Int, cassette2: Int, newName: String): Machine
+    machineAction(
+      deviceId: ID!
+      action: MachineAction!
+      cassette1: Int
+      cassette2: Int
+      newName: String
+    ): Machine
     machineSupportLogs(deviceId: ID!): SupportLogsResponse
     serverSupportLogs: SupportLogsResponse
     setCustomer(customerId: ID!, customerInput: CustomerInput): Customer
     saveConfig(config: JSONObject): JSONObject
     createPairingTotem(name: String!): String
     saveAccounts(accounts: JSONObject): JSONObject
+    deleteBlacklistRow(cryptoCode: String, address: String): [String]
```

Comment:
Don't we want cryptoCode and address to be required here as well? Is there are reason why this returns an array?